Compilation error

@Abestanis: in AutoUpdater::parseDownloadUrl() we have the following code:

```
    for (auto assetInfo : assets) {
        QString name = assetInfo["name"].toString();    
```

This doesn't compile:
```
autoupdater.cpp
C:\Users\test\Builds\src\autoupdater.cpp(165): error C2676: binary '[': 'QJsonValue' does not define this operator or a conversion to a type acceptable to the predefined operator
C:\Users\test\Builds\src\autoupdater.cpp(165): error C2228: left of '.toString' must have class/struct/union
C:\Users\test\Builds\src\autoupdater.cpp(167): error C2676: binary '[': 'QJsonValue' does not define this operator or a conversion to a type acceptable to the predefined operator
C:\Users\test\Builds\src\autoupdater.cpp(167): error C2228: left of '.toString' must have class/struct/union
C:\Users\test\Builds\src\autoupdater.cpp(169): error C2676: binary '[': 'QJsonValue' does not define this operator or a conversion to a type acceptable to the predefined operator
C:\Users\test\Builds\src\autoupdater.cpp(169): error C2228: left of '.toVariant' must have class/struct/union
C:\Users\test\Builds\src\autoupdater.cpp(169): error C2228: left of '.toULongLong' must have class/struct/union
NMAKE : fatal error U1077: 'C:\PROGRA~2\MICROS~1.0\VC\bin\cl.exe' : return code '0x2'

```

Assets is QJsonArray, and thus assetInfo is QJsonValue. It doesn't have bracket method. Isn't it supposed to be assetInfo.toObject()["name"] ?

--- a/src/autoupdater.cpp
+++ b/src/autoupdater.cpp
@@ -162,11 +162,12 @@ qulonglong AutoUpdater::parseDownloadUrl(const QJsonArray &assets, const QString
     const QString fileEnding = is64Bit ? "x64.exe" : "x86.exe";
 #endif
     for (auto assetInfo : assets) {
-        QString name = assetInfo["name"].toString();
+        const QJsonObject assetsInfoObject = assetInfo.toObject();
+        QString name = assetsInfoObject["name"].toString();
         if (!name.isNull() && name.endsWith(fileEnding)) {
-            downloadUrl = assetInfo["browser_download_url"].toString();
+            downloadUrl = assetsInfoObject["browser_download_url"].toString();
             if (downloadUrl.isValid()) {
-                return assetInfo["size"].toVariant().toULongLong();
+                return assetsInfoObject["size"].toVariant().toULongLong();
             }
         }
     }

